bottles/backend/managers/manager.py

```diff
@@ -370,7 +370,7 @@ def check_runners(self, install_latest: bool = True) -> bool:
         tmp_runners = [x for x in self.runners_available if not x.startswith('sys-')]
 
         if len(tmp_runners) == 0 and install_latest:
-            logging.warning("No runners found.")
+            logging.warning("No managed runners found.")
```

Comment:
Doesn't seem relevant?

Reply:
> Doesn't seem relevant?

This log confused me becuase it found sys runner but still says "No runner found", so I made it more clear.
I can revert it if you like.

Reply:
It would be better in a separate PR.